docs/DEPLOYMENT.md

```diff
@@ -0,0 +1,21 @@
+# Deployment Guide
+
+1. Login to the data team's server:
+    1. Run: `gcloud compute ssh --zone "asia-east1-b" "data-team" --project "pycontw-225217"`
+    2. Services:
+        * ETL: `/home/zhangtaiwei/pycon-etl`
+        * Metabase is located here: `/mnt/disks/data-team-additional-disk/pycontw-infra-scripts/data_team/metabase_server`
+
+2. Pull the latest codebase to this server: `sudo git pull`
```

Comment:
`sudo` looks dangerous. Just want to confirm whether it's required

Reply:
This is a issue, project path should be move to another path.

Reply:
got it